code/game/machinery/vending/vendor_types/requisitions.dm

```diff
@@ -289,11 +297,12 @@
 		list("Shotgun Shell Box (Buckshot x 100)", 0, /obj/item/ammo_box/magazine/shotgun/buckshot, VENDOR_ITEM_REGULAR),
 		list("Shotgun Shell Box (Flechette x 100)", 0, /obj/item/ammo_box/magazine/shotgun/flechette, VENDOR_ITEM_REGULAR),
 		list("Shotgun Shell Box (Slugs x 100)", 0, /obj/item/ammo_box/magazine/shotgun, VENDOR_ITEM_REGULAR),
-		list("Shotgun Shell Box (16g) (Breaching x 120)", 1, /obj/item/ammo_box/magazine/shotgun/light/breaching, VENDOR_ITEM_REGULAR),
+		list("Shotgun Shell Box (16g) (Breaching x 120)", 0, /obj/item/ammo_box/magazine/shotgun/light/breaching, VENDOR_ITEM_REGULAR),
 		list("Magazine Box (88 Mod 4 AP x 16)", 0, /obj/item/ammo_box/magazine/mod88, VENDOR_ITEM_REGULAR),
 		list("Magazine Box (SU-6 x 16)", 0, /obj/item/ammo_box/magazine/su6, VENDOR_ITEM_REGULAR),
 		list("Magazine Box (VP78 x 16)", 0, /obj/item/ammo_box/magazine/vp78, VENDOR_ITEM_REGULAR),
 		list("Magazine Box (XM51 x 8)", 0, /obj/item/ammo_box/magazine/xm51, VENDOR_ITEM_REGULAR),
+		list("Rounds Box (.458 SOCOM x 300)", 0, /obj/item/ammo_box/magazine/lever_action/xm88, VENDOR_ITEM_REGULAR),
```

Comment:
Of note how this box counts handfuls is really weird in its examine. But not necessarily a problem you created. I can't see any *actual* problem with the ammo counts but its just it suggests it contains 540 rounds since it claims it has 60 handfuls, and each handful should have 9 rounds. But in actuality it contains 33 full handfuls and 1 handful of 3.
![socom 300](https://github.com/user-attachments/assets/12b7769e-86e9-423a-8cc1-f6ab8a1152b9)
![image](https://github.com/user-attachments/assets/bcd86ae9-2161-4dd2-a2d1-20b51edb0c67)

If you want to fix this, just fix the implementation of `/obj/structure/magazine_box/get_examine_text(mob/user)` to divide by `max_rounds` instead of just always `5` https://github.com/cmss13-devs/cmss13/blob/e87c7683781347c96705a8a68c6477e664ba24d9/code/modules/projectiles/ammo_boxes/box_structures.dm#L104

Reply:
fixed